src/js/calculator/home_care_client_fee/_translations.js

```diff
@@ -177,24 +172,19 @@ const translations = {
     en: 'If both spouses are clients of continuous and regular home care with a monthly fee, the fee is divided between them in accordance with their service and care plans.',
   },
   receipt_shopping_service_math_single: {
-    fi: '1 kauppatoimitus viikossa, kuukausihintaan on laskettu 4 toimitusta.',
-    sv: '1 butiksleverans per vecka, 4 leveranser ingår i månadspriset.',
-    en: '1 grocery delivery per week; the monthly fee includes 4 deliveries.',
+    fi: '1 kauppatoimitus viikossa, kuukausihintaan on laskettu 4 toimitusta.',
+    sv: '1 butiksleverans per vecka, 4 leveranser ingår i månadspriset.',
+    en: '1 grocery delivery per week; the monthly fee includes 4 deliveries.',
   },
   receipt_shopping_service_math_multiple: {
-    fi: '${delivery_count_per_week} kauppatoimitusta viikossa, kuukausihintaan on laskettu ${delivery_count_per_month} toimitusta.',
-    sv: '${delivery_count_per_week} butiksleveranser per vecka, i månadspriset ingår ${delivery_count_per_month} leveranser.',
-    en: '${delivery_count_per_week} grocery deliveries per week; the monthly fee includes ${delivery_count_per_month} deliveries.',
+    fi: '1 kauppatoimitus viikossa, kuukausihintaan on laskettu 4 toimitusta.',
```

Comment:
Now this seems duplicate for the above translation, why not remove this translation altogether?

Reply:
Also, this translation is no longer referenced in any of our code because you remove it from: `home_care_client_fee.js` in this PR.

Reply:
Good catch :D removed!

Reply:
👍